### DISCUSSION TITLE
Distributed network

### DISCUSSION THREAD

**Discussion by Haw984** [HewlettPackard/PacketRusher#140] [General]
Hi,
Is it possible to create a distributed network, where I can use one gNB on one container and one UE on another?

Thanks a lot!

**Comment by linouxis9** [comment]
Hi @Haw984,

Sadly, it is not yet possible with PacketRusher. However, you can spawn as much containers you want, with each containing a gNB/UE combo if you would like!

Cheers,
Valentin
